### PR TITLE
VR-3327 remove id from folder element

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/entities/versioning/InternalFolderElementEntity.java [hunk 1]
```diff
@@ -23,13 +22,10 @@ public InternalFolderElementEntity(
...
 
   @Id
-  @GeneratedValue(strategy = GenerationType.IDENTITY)
-  @Column(name = "id", updatable = false, nullable = false, columnDefinition = "UNSIGNED")
-  private Long id;
-
   @Column(name = "folder_hash", nullable = false)
   private String folder_hash;
 
+  @Id
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/entities/versioning/InternalFolderElementEntity.java]: I think the ID should be (folder sha, element name). A given folder can't have multiple elements with the same name. The multiple elements could be a copy of each other though (which would make the sha the same).

Comment (evgeniilezhnin) [reply backend/src/main/java/ai/verta/modeldb/entities/versioning/InternalFolderElementEntity.java]: fixed

**Comment by evgeniilezhnin** [comment]
so as you said that we can drop the table: I think it can be done in liquibase. Pushed a change for it

**Comment by evgeniilezhnin** [comment]
verified on le.dev

**Pull request merged**